app/services/selection/selection.ts

```diff
@@ -453,10 +453,18 @@ export class Selection implements ISelection {
     }
   }
 
+  /**
+   * A selection is considered visible if at least 1 item
+   * in the selection is visible.
+   */
   isVisible() {
-    return !this.getItems().find(item => !item.visible);
+    return this.getItems().find(item => item.visible);
```

Comment:
The returning type now is `Item`. Must be `boolean`

Reply:
Good call.